tests/test_distribution.py

```diff
@@ -0,0 +1,56 @@
+""" Tests for the distribution container """
+import json
+import os
+
+import pytest
+from pytest_container import OciRuntimeBase
+
+from bci_tester.data import DISTRIBUTION_CONTAINER
+from bci_tester.runtime_choice import DOCKER_SELECTED
+
+CONTAINER_IMAGES = [DISTRIBUTION_CONTAINER]
+
+
+@pytest.mark.skipif(
+    DOCKER_SELECTED,
+    reason="docker does not support the same volume flags as podman (missing CHOWN and NOEXEC), e.g. https://github.com/moby/moby/issues/7054",
+)
```

Comment:
Given that the container image itself is unusable with docker, I would suggest to add this mark directly to the container.

Reply:
That makes perfect sense, done. Thanks for the suggestion.